src/actions/ActionHelpers.ts

```diff
@@ -0,0 +1,533 @@
+// eslint-disable-next-line import/no-unresolved
+import { Frame, IHasName, Layout, ListVariable, Page, StylekitChacterStylesController, StylekitColorsController, StylekitParagraphStylesController, Variable, VariableValue } from 'grafx-studio-actions';
+
+/**
+ * Whether the variable is the trigger for this action
+ * @param variable optional variable name
+ * @returns true if it was the trigger, otherwise false
+ */
+export function isVariableTriggered(variable?: string | Variable): boolean {
+    const changedVariable = triggers.changedVariable;
+
+    if (!changedVariable){
+        return false;
+    }
+
+    let variableName = variable;
+
+    if (variable && (variable as Variable).name) {
+        variableName = (variable as Variable).name;
+    }
+
+    if (typeof variableName !== 'string') {
+        throw new Error('variable argument should be of type "string" but was ' + typeof variableName);
+    }
+
+   return changedVariable.name.toLocaleLowerCase() === variableName.toLocaleLowerCase();
+}
+
+/**
+ * Gets the triggered variable
+ */
+export function getTriggeredVariable(): Variable {
```

Comment:
See my comment on `getVariable`, clients who are using helper functions are not going to be thinking in objects. They will get back something strange.

I suggest `getTriggeredVariableName`